packages/yourloops/pages/signup/signup-profile-form.tsx

```diff
@@ -175,7 +175,7 @@ function SignUpProfileForm(props: SignUpFormProps): JSX.Element {
         id="firstname"
         className={classes.TextField}
         margin="normal"
-        label={t("firstname")}
+        label={t("firstName")}

```

Comment:
```suggestion
        label={t("firstname")}
```
Yourloops tr key is in lowercase